src/styles/features/mcp-modal.css

```
@@ -437,33 +437,27 @@
+  animation: spin 0.8s linear infinite;
   margin: 0 auto 12px;
 }
 
-@keyframes mcp-spin {
-  to {
-    transform: rotate(360deg);
-  }
-}
-
-.mcp-error {
+@keyframes.mcp-error {
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@keyframes.mcp-error`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)